pkgs/by-name/me/memray/package.nix

```diff
@@ -1,11 +1,11 @@
 {
   lib,
-  elfutils,
+  python3Packages,
   fetchFromGitHub,
+  pkg-config,
+  elfutils,
   libunwind,
   lz4,
-  pkg-config,
-  python3Packages,
```

Comment:
Please keep this alphabetically ordered after `lib`.

Reply:
Reverted :)